src/ngraph/frontend/onnx_import/op/prelu.cpp

```diff
@@ -43,44 +33,9 @@ namespace ngraph
                 NodeVector prelu(const Node& node)
```

Comment:
In cpp files in ngraph we do not use nested namespaces like this, we change `NodeVector prelu(...)` to `NodeVector onnx_import::op::set1::prelu(...)`. This removes all of the 16 extra spaces of indenting.

Reply:
We have this convention in the entire `onnx_import` frontend, so I don't want to change it one file at a time. At some point we may decide to change it globally.